libraries/ESP8266EthernetWebServer/library.properties

```diff
@@ -0,0 +1,9 @@
+name=ESP8266EthernetWebServer
+version=1.0
+author=Ivan Grokhotkov
+maintainer=Ivan Grokhtkov <ivan@esp8266.com>
```

Comment:
@burner- What's your plan for maintaining this library? Do you plan to update it with all the changes/fixes which the original ESP8266WebServer library receives?

If not, may I suggest that we take a look at the original library and add a bit of generics so that it can work both with ESP8266WiFi and Ethernet libs? I feel this may be a more maintainable approach for the longer term.
In any case, thank you for the contribution and would love to hear what you think of this.